packages/showcases/core/csf/components/indicators/tag.csf.js

```diff
@@ -48,4 +50,13 @@ export const argTypes = {
     defaultValue: '#',
     control: { type: 'text' },
   },
+  icon: {
+    type: { name: 'string', required: false },
+    description: 'The start icon of the tag.',
+    defaultValue: 'leaf-fill',
+    control: {
+      type: 'select',
+      options: ['', ...Object.keys(vitamixIconsList)],
+    },
+  },
```

Comment:
Did you verify that by adding this it works also for React & Svelte?

Reply:
I added the icon for the svelte part !